loadbalancer.cfhighlander.rb

```diff
@@ -17,4 +17,4 @@
     subnet_parameters({'compute'=>{'name'=>'Compute'}}, maximum_availability_zones) if defined?(loadbalancer_scheme) && loadbalancer_scheme == 'internal'
     OutputParam component: 'vpc', name: "VPCId"
```

Comment:
@Guslington OutputParams are now deprecated also. If there is component with output `VPCId` in outer component, this parameter gets autowired - take a look at https://github.com/theonestack/cfhiglhander-params-demo for more details.